next.config.js

```diff
@@ -14,3 +15,20 @@ module.exports = withBundleAnalyzer({
     ]
   },
 })
+
+const SentryWebpackPluginOptions = {
+  // Additional config options for the Sentry Webpack plugin. Keep in mind that
+  // the following options are set automatically, and overriding them is not
+  // recommended:
+  //   release, url, org, project, authToken, configFile, stripPrefix,
+  //   urlPrefix, include, ignore
+
+  silent: process.env.NODE_ENV !== 'production', // Suppresses all logs in envs other than production
+  deploy: {
+    env: process.env.VERCEL_ENV,
```

Comment:
This is the only thing I added myself so we can classify errors per environment. [`VERCEL_ENV` is exposed by Vercel](https://vercel.com/docs/environment-variables#system-environment-variables) so this will be either `production`, `preview`, or `development`.

Reply:
That seems handy, you can see which environment is broken. 😄